coordinator/setup.py

```diff
@@ -374,8 +374,9 @@ def parse_version(root, **kwargs):
     },
     setup_requires=[
         "setuptools_scm>=5.0.0",
-        "grpcio<=1.43.0,>=1.40.0",
-        "grpcio-tools<=1.43.0,>=1.40.0",
+        "grpcio",
+        "grpcio-tools",
```

Comment:
If there's no version control for grpcio and grpcio-tools, the build process may failure in user's development environemnt as well, (imaging there's are some staled version installed)